assets/src/dashboard/app/views/templates/components.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+export const ContentContainer = styled.div`
+  ${({ theme }) => `
+    padding: 0 ${theme.pageGutter.large.desktop}px;
+
+    @media ${theme.breakpoint.tablet} {
+      padding: 0 ${theme.pageGutter.large.tablet}px;
+    }
+
+    @media ${theme.breakpoint.largeDisplayPhone} {
+      padding: 0;
+    }
+  `}
+`;
+
+export const ColumnContainer = styled.section`
+  ${({ theme }) => `
+    display: flex;
+
+    @media ${theme.breakpoint.largeDisplayPhone} {
+      display: block;
+    }
+  `}
+`;
+
+export const Column = styled.div`
+  ${({ theme }) => `
+    width: 50%;
+
+    & + & {
+      padding-left: ${theme.pageGutter.large.desktop}px;
+    }
+
+    @media ${theme.breakpoint.tablet} {
+      & + & {
+        padding-left: ${theme.pageGutter.large.tablet}px;
+      }
+    }
+
+    @media ${theme.breakpoint.largeDisplayPhone} {
+      width: 100%;
+    }
+  `}
+`;
+
+export const DetailContainer = styled.div`
+  padding: 40px 20px;
+`;
+
+export const Title = styled.h2`
+  ${({ theme }) => `
+    margin: 0 0 5px;
+    font-family: ${theme.fonts.heading2.family};
+    font-size: ${theme.fonts.heading2.size};
+    font-weight: ${theme.fonts.heading2.weight};
+    line-height: ${theme.fonts.heading2.lineHeight};
+    color: ${theme.colors.gray900};
+  `}
+`;
+
+export const ByLine = styled.p`
+  ${({ theme }) => `
+    margin: 0 0 20px;
+    font-family: ${theme.fonts.body1.family};
+    font-size: ${theme.fonts.body1.size};
+    font-weight: ${theme.fonts.body1.weight};
+    line-height: ${theme.fonts.body1.lineHeight};
+    color: ${theme.colors.gray400};
+  `}
+`;
+
+export const Text = styled.p`
+  ${({ theme }) => `
+    margin: 0 0 20px;
+    font-family: ${theme.fonts.body2.family};
+    font-size: ${theme.fonts.body2.size};
+    line-height: 22px;
+    letter-spacing: 0.015em;
+    color: ${theme.colors.gray600};
+  `}
+`;
+
+export const MetaDataContainer = styled.fieldset`
```

Comment:
this is so minor, but we should be consistent with metadata vs metaData I feel like. You have `metadata` in the apiProvider changes above.

Reply:
Yes! Good catch :D